app/components/admin/users/tools/credits_component.html.erb

```diff
@@ -0,0 +1,144 @@
+<section
+  class="crayons-card p-3 js-component"
+  data-controller="users--tools--ajax"
+  data-action="ajax:success@document->users--tools--ajax#success ajax:error@document->users--tools--ajax#error">
+  <header>
+    <h3><%= link_to "← Credits", admin_user_tools_path(@user),
+                    remote: true,
+                    data: { action: "ajax:success@document->user#replacePartial" },
+                    class: "crayons-title fs-l" %></h3>
+  </header>
+
+  <%# Add user credits %>
+  <%= form_with url: admin_user_tools_credits_path(@user),
+                method: :post,
+                remote: true,
+                class: "my-3",
+                id: :credits_add do |f| %>
+    <fieldset>
+      <legend class="fs-base fw-medium mb-3"><%= pluralize(@user_unspent_credits_count, "available user credit") %></legend>
+
+      <div class="crayons-field wb-2">
+        <%= f.label "Add credits", class: "crayons-field__label", id: "add-credits" %>
+      </div>
+
+      <div class="crayons-field form-group w-50">
+        <%= f.number_field "credits[count]",
+                           id: nil,
+                           in: 1...10_000,
+                           required: true,
+                           class: "crayons-textfield",
+                           "aria-labelledby": "add-credits" %>
+        <%= f.text_field "credits[note]",
+                         id: nil,
+                         placeholder: "Why are you adding these credits?",
+                         required: true,
+                         class: "crayons-textfield",
+                         "aria-labelledby": "add-credits" %>
```

Comment:
At the moment, both of these fields have the same label, when they're for different things. I think we can just add an aria-label to each 🙂

Reply:
Oh cool, I was trying to have them be described by the same thing but you're right, they have a different purpose

Reply:
Yeah the nice thing about having the "Add credits" bit as the fieldset's `legend` is that it'll still be linked to these inputs, just as a descriptive "here's what this group does", rather than the more granular "here's what this input does"... if that makes sense 😄